src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -184,9 +184,7 @@ function bindCopyButton() {
 }
 
 function bindCopyEvents() {
-
     $('#copyTableModal >tbody>tr').on('click', function(e) {
-        e.preventDefault();
 
         var currentRow = $(this);
```

Comment:
Would be good to seperate the variable and the following block of flow control code.

Additionally, try to prefix jQuery wrapped objects with $, like $currentRow, as per https://docs.google.com/document/d/1gZ6WG6HBTJYHAtVkz9kzi_SUuzfXqzO-SvFnLuag2xM/pub?embedded=true -> https://github.com/airbnb/javascript/tree/master/es5